chapters/annotations.tex

```diff
@@ -1196,6 +1197,8 @@ \section{Annotations for the Graphical User Interface}\doublelabel{annotations-f
                   groupImage="modelica://MyPackage/Resources/Images/switch.png",
                   connectorSizing = false));
 \end{lstlisting}
+For a short replaceable class definition only the fields \lstinline!tab!, \lstinline!group!, \lstinline!enable! and
+\lstinline!groupImage! are supported.
```

Comment:
To me, this is not a matter of supporting something, it is a matter of what is applicable.  This would be better reflected with _allowed_:
```suggestion
\lstinline!groupImage! are allowed.
```